ui/page/home/view.jsx

```diff
@@ -193,6 +194,26 @@ function HomePage(props: Props) {
 
   return (
     <Page>
+      {(authenticated || !IS_WEB) && !subscribedChannels.length && (
```

Comment:
could this check pass while channels are loading?

Reply:
The subscribed channels are repopulated before this page is rendered.